openmdao/error_checking/check_config.py

```diff
@@ -361,6 +361,20 @@ def _check_comp_has_no_outputs(problem, logger):
         logger.warning(''.join(["The following Components do not have any outputs:\n"] + msg))
 
 
+def _check_auto_ivc_warnings(problem, logger):
+    """
+    Issue a logger warning if any components have conflicting attributes.
+
+    Parameters
+    ----------
+    problem : <Problem>
+        The problem being checked.
+    """
+    if hasattr(problem.model, "_auto_ivc_warnings"):
+        for i in problem.model._auto_ivc_warnings:
+            simple_warning(i)
```

Comment:
instead of using a simple_warning here, you should follow what the other check functions do, which is logger.warning(...)